### PR TITLE
Implement twitter

### PR DESCRIPTION
**Pull request by khamzahabibullah**
Implpemented dropdown on the home page

### CODE REVIEW

**Review by Sajakhtar — APPROVED** [review]

File: app/controllers/topics_controller.rb [hunk 1]
```diff
@@ -1,7 +1,56 @@
+require 'date'
+require 'json'
+require 'typhoeus'
+
 class TopicsController < ApplicationController
   skip_before_action :authenticate_user!, only: :show
 
   def show
     @topic = Topic.find(params[:id])
+    @recent_search = twitter_recent_search['data']
```
Comment (andrerferrer) [inline app/controllers/topics_controller.rb:10]: arrays should be named in plural for easier comprehension

File: app/controllers/topics_controller.rb [hunk 2]
```diff
@@ -1,7 +1,56 @@
+require 'date'
+require 'json'
+require 'typhoeus'
```
Comment (andrerferrer) [inline app/controllers/topics_controller.rb:3]: All of this should be required by bundler already. Please remove and try it out. It should work just as fine

File: app/controllers/topics_controller.rb (hunk 1 above)
Comment (andrerferrer) [reply app/controllers/topics_controller.rb:10]: maybe we can call it `@tweets`?

**Review by andrerferrer — COMMENTED** [review]
Please check commentes @Sajakhtar @khamzahabibullah

**Comment by octaserban** [comment]
Already taken care of yesterday 24-Aug. Card archived

**Pull request merged**